kv/etcd/options.go

```diff
@@ -98,6 +104,7 @@ type options struct {
 	watchChanCheckInterval time.Duration
 	watchChanResetInterval time.Duration
 	watchChanInitTimeout   time.Duration
+	watchStartRevision     int64
```

Comment:
So by default we watch with revision 0 presumably?

Reply:
We won't even bother setting the value if the user doesn't specify a non-zero start revision. 0 default is what etcd does under the hood as well, if the request has start revision 0 the server just assumes "now"